tests/integration/functional/artifacts/test_utils.py

```diff
@@ -124,6 +124,11 @@ def test_log_artifact_metadata_existing(clean_client):
         artifact_name="meaning_of_life",
         artifact_version="1",
     )
+    log_artifact_metadata(
```

Comment:
can you also tests list of strings and floats to make sure?

Reply:
Extended